### PR TITLE
bots: Only run Kube namespace specific tasks in that namespace

### PR DESCRIPTION
**Pull request by stefwalter**
If a task has a kubernetes namepsace DNS URL, then only run inside the
relevant Kubernetes project/namespace

### CODE REVIEW

**Comment by stefwalter** [comment]
Tested manually with running ```oc rsh ...``` and ```bots/issue-scan```

File: bots/issue-scan [hunk 1]
```diff
@@ -72,21 +76,34 @@ def main():
...
         elif not redhat and contains_any(result, REDHAT_TASKS):
             continue
         elif contains_any(result, WINDOWS_TASKS):
             continue
+        elif contains_any(result, KUBERNETES_TASKS):
+            if not namespace:
+                continue
+            url = namespace + KUBERNETES_TASKS[0]
+            if url not in result:
+                continue
```
Comment (martinpitt) [inline bots/issue-scan]: It might be useful for debugging to log about this here?

Comment (stefwalter) [reply bots/issue-scan]: What about all the others above? Should they also have log output?

Comment (martinpitt) [reply bots/issue-scan]: Good point, maybe. But I figure we can introduce the logging when that actually becomes a problem.

**Pull request merged**